.github/workflows/lint.yml

```diff
@@ -12,8 +12,9 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2.4.0
       - name: Run Super-Linter
-        uses: github/super-linter@v4.8.2
+        uses: github/super-linter@v4.8.3
```

Comment:
In these type of PRs I often see this change. Could the [latest](https://github.com/github/super-linter/wiki/Releases-and-how-they-work) tag be of any use here?

Reply:
Tempting, the drawback is it can start failing suddenly when new version is released

Reply:
Agreed. Let's stick with specific version and let dependabot handle upgrades.